core/src/main/java/io/grpc/Channel.java

```diff
@@ -34,15 +34,14 @@
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * A Channel provides an abstraction over the transport layer that is designed to be consumed
- * by stub implementations. Channel and its associated types {@link ClientCall} and
- * {@link ClientCall.Listener} exchange parsed request and response objects whereas the
- * transport layer only works with serialized data.
+ * A virtual connection to an endpoint, to perform RPCs. A channel is free to have zero or many
+ * actual connections to the endpoint based on configuration, load, etc. Applications are generally
```

Comment:
Yeah. Done. I'm okay with the language I added, but I would be open to improvements.